bundles/org.openhab.core.automation.module.script/src/main/java/org/openhab/core/automation/module/script/ScriptTransformationService.java

```diff
@@ -117,6 +117,7 @@ public void deactivate() {
 
         // cleanup script engines
         scriptCache.values().forEach(this::disposeScriptRecord);
+        scriptCache.clear();
```

Comment:
What is your idea behind that? After `deactivate` is called the service is no longer available and it can only be activated by calling the constructor (i.e. creating a new object which would start with an empty `Map`).